python/ray/serve/_private/deployment_graph_build.py

```diff
@@ -225,13 +225,20 @@ def replace_with_handle(node):
         # TODO: (jiaodong) Need to capture DAGNodes in the parent node
         parent_deployment_node = other_args_to_resolve[PARENT_CLASS_NODE_KEY]
 
+        parent_class = parent_deployment_node._deployment._func_or_class
+        method = getattr(parent_class, dag_node._method_name)
+        if "return" in method.__annotations__:
+            other_args_to_resolve["return_type_annotation"] = str(
```

Comment:
I return string here because deployment nodes are serialized to and from json, and `type` is not json serializable.